### PR TITLE
added time line for fund 7

### PR DESCRIPTION
**Pull request by vasanthaRaghu**
I have removed the Fund 6 upon us and added the Fund 7 Timelines.

### CODE REVIEW

**Comment by mwojtera** [comment]
Thank you for your contribution. I will take a look at it today!

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@js-joda/core": "^3.2.0",
+    "@vuepress/plugin-container": "^2.0.0-beta.27",
```
Comment (mwojtera) [inline package.json:33]: Why was this package added?

**Review by mwojtera — CHANGES_REQUESTED** [review]
Hi, I did some minor improvements. There is just one question that I would like you to answer, before I can approve the PR. (It's in the conversation about package.json)

File: package.json (hunk 1 above)
Comment (vasanthaRaghu) [reply package.json:33]: oh. tried to include a container on the page. It's not required will remove it

**Comment by vasanthaRaghu** [comment]
Added a link to the Fund 6 CA and vCA review statistics and improved the layout of the Graphs from Fund 5 Voting

**Review by mwojtera — APPROVED** [review]

**Comment by mwojtera** [comment]
I have a problem running `npm run build`. I get.
`Error evaluating async computed property: Error: Request failed with status code 504
    at createError (server-bundle.js:4844:15)
    at settle (server-bundle.js:53108:12)
    at IncomingMessage.handleStreamEnd (server-bundle.js:60309:11)
    at IncomingMessage.emit (events.js:412:35)
    at endReadableNT (internal/streams/readable.js:1334:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)`

Can you verify it's building correctly for you?

**Comment by mwojtera** [comment]
Can you check which version of node and npm you are building against? Maybe this is the problem

**Pull request merged**